src/NumericalAlgorithms/Spectral/Spectral.hpp

```diff
@@ -87,7 +87,13 @@ std::ostream& operator<<(std::ostream& os, const Basis& basis);
  * basis), and thus do not implement differentiation matrices, integration
  * weights, and interpolation matrices.
  */
-enum class Quadrature { Gauss, GaussLobatto, CellCentered, FaceCentered };
+enum class Quadrature {
+  Gauss,
+  GaussLobatto,
+  CellCentered,
+  FaceCentered,
+  SphericalHarmonic
```

Comment:
Can we describe better how Spherepack distributes collocation points? I think it's just equiangular, so maybe name this `Equiangular`?

Reply:
I'm fine with this, but I don't know if equiangular is correct. @scheel?

Reply:
So this sounds to me you can add `Equiangular` here for the \phi direction, and we can use `Gauss` for the \theta direction?